frontend/src/components/MapView/LeftPanel/ChartsPanel/ChartSection/index.tsx

```diff
@@ -49,8 +50,8 @@ function generateDateStrings(startDate: Date, endDate: Date) {
   const result = [];
   const interval = [1, 11, 21];
   const currentDate = new Date(startDate);
-  currentDate.setHours(10);
-  endDate.setHours(12);
+  currentDate.setHours(10, 0, 0, 0);
```

Comment:
not from this PR but curious why we set it to 10 here?

Reply:
It is like that so in the `while` bellow the current is before endDate

Reply:
the while loop uses a `<=` so it would fo through anyway for the first day, no?

Reply:
I am adding a test for this function (on the tests PR #1090 ). I believe it has to do with daylight savings. I will confirm and add a comment also

Reply:
seems to work ok with 12, so i set it to 12